doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1577,19 +1577,13 @@ succeeds, and results in an error otherwise.
    .. exn:: Not equal (due to universes).
       :undocumented:
 
-.. tacn:: constr_eq @one_term @one_term
+   .. tacn:: constr_eq @one_term @one_term
 
-   Like :tacn:`constr_eq_strict`, but may add constraints to make universes equal.
+      Like :tacn:`constr_eq_strict`, but may add constraints to make universes equal.
```

Comment:
A trifle curious to have this one second, which sounds like the most basic option.

Reply:
Users probably do not expect that an equality test would have side effects, which is probably the reason why this is not the behavior of `constr_eq`.

Reply:
constr_eq does have side effects, constr_eq_strict is the non side effect one

Reply:
Right, I was confused when writing my answer. I meant to say: this is probably why `constr_eq_strict` is put forward in the doc.